src/Sylius/Behat/Resources/config/services/contexts/ui.xml

```diff
@@ -104,6 +104,12 @@
             <argument type="service" id="sylius.behat.shared_security" />
         </service>
 
+        <service id="sylius.behat.context.ui.admin.managing_payments" class="Sylius\Behat\Context\Ui\Admin\ManagingPaymentsContext">
```

Comment:
```suggestion
        <service id="Sylius\Behat\Context\Ui\Admin\ManagingPaymentsContex" >
```
?

Reply:
convention in file src/Sylius/Behat/Resources/config/services/contexts/ui.xml is made lowercase and using id and class separately